pkgs/shells/zsh/antidote/default.nix

```
@@ -0,0 +1,29 @@
+
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
```

```suggestion
{ lib, stdenvNoCC, fetchFromGitHub }:

stdenvNoCC.mkDerivation rec {
```
Since this derivation doesn't involve any building and solely entails file copying, it would be more appropriate to utilize `stdenvNoCC` instead.